internal/kafka/internal/services/kafka.go

```diff
@@ -54,6 +54,12 @@ var kafkaManagedCRStatuses = []string{
 	constants.KafkaRequestStatusResuming.String(),
 }
 
+var statusesOfKafkaThatAreNotActive = []string{
```

Comment:
should this also include `KafkaRequestStatusSuspending`?

Reply:
As long as the Kafka hasn't been suspended, I think we can continue to reconcile the certs.